.github/workflows/ci.yml

```diff
@@ -13,9 +13,9 @@ jobs:
           - java: 11
             scala: 2.12.18
           - java: 11
-            scala: 2.13.11
+            scala: 2.13.13
           - java: 11
-            scala: 3.3.0
+            scala: 3.4.0
```

Comment:
This is not an LTS release. Let's revert this.

Reply:
Unfortunately, this PR won't work with scala < 3.4 because of https://github.com/scala/scala3/issues/16630

They mentioned backporting to 3.3.1-RC1 but seems like that never happened. It was also mentioned briefly here: https://github.com/lloydmeta/enumeratum/issues/300#issuecomment-1374313401

Reply:
Understood. That is a blocker indeed.

Reply:
Makes sense. I tested this out using scala 3.3.4-RC4 and it does build succesfully with that version. So once 3.3.4 gets released I'll udpate this PR.

Reply:
this is fixed now. 3.3.4 was released.